src/renderer/index.html

```diff
@@ -20,6 +20,6 @@
     </style>
 </head>
 <body>
-    <div id="app"><p class="splash">Scratch Desktop is loading...</p></div>
+    <div id="app"><p class="splash">Scratch is loading...</p></div>
```

Comment:
This file is intended to show up immediately, while we wait for other files to load. It doesn't load any external resources itself, so it can't access `productName`.